compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -741,6 +742,9 @@ template <> bool InstanceNormPattern::match<InstanceNormPattern::PatternVersion:
   CHECK_OR_FALSE(rsqrt == rsqrt_should_be);
   CHECK_OR_FALSE(mean_of_ifm == mean_of_ifm_should_be);
 
+  // make clone for shared beta node that gets reshaped in reshape_gamma_beta()
+  const_as_beta = luci::clone(const_as_beta);
```

Comment:
Could you add an origin to this node?

Reply:
`luci::clone` will also copy the name of a node. So, there can be nodes with the same name. This will break the invariant of luci graph, but we don't have any code to handle such a case.. That is the out of the scope of this PR, so let's consider this later.